libs/generators/src/lib/generators/generators-menu/generators-menu.component.html

```diff
@@ -1,7 +1,11 @@
 <mat-nav-list>
-    <div mat-list-item *ngFor="let menuItem of menuItems">
-        <a mat-button
-           class="menu-item" 
-           color="primary">{{menuItem}}</a>
-    </div>
+  <div mat-list-item *ngFor="let menuItem of menuItems">
+    <a
+      mat-button
+      class="menu-item"
+      color="primary"
+      (click)="navigate(menuItem)"
```

Comment:
Maybe use [routerLink](https://angular.io/api/router/RouterLink#relative-link-paths) instead of click